styles/new/base.css

```
@@ -41,3 +41,50 @@ ol {
     text-neutral-900
     dark:text-white;
 }
+
+table {
+  @apply w-full 
+  border-separate 
+  border-spacing-0 
+  rounded
+  border
+  border-neutral-200 
+  text-left 
+  dark:border-neutral-800;
```

Figma prototype use 14px font size.
```suggestion
  dark:border-neutral-800
  text-sm;
```